### PR TITLE
feat: remove old anvil catalog build and code (#2647)

### LINKED ISSUE #2647: Remove old AnVIL catalog build and code
PR this to  the noopdog/2624-UCSC-AWS branch
If you want to block the build first and do the code in a later ticket, that's fine just rename this ticket and create a new one.

### PR DESCRIPTION
**Pull request by frano-m**
Closes #2647

### CODE REVIEW

File: content/faq/resources-for-anvil-users.md [hunk 1]
```diff
@@ -16,17 +16,28 @@ The current list of large-scale datasets which will be available through AnVIL a
 - eMERGE
 - CSER (coming soon)
 
-The AnVIL Data Dashboard provides a real time view of which data sets and individual projects are available on [AnVIL](/data).
+The AnVIL Data Dashboard provides a real time view of which data sets and individual projects are available
+on [AnVIL](https://anvilproject.org/data/consortia).
```
Comment (frano-m) [inline content/faq/resources-for-anvil-users.md:20]: Any content "refactored" is due to a change in the internal link from `/data` to `https://anvilproject.org/data/consortia` for AnVIL and `https://anvilproject.org/ncpi/data/studies` for NCPI - and also prettier updating the markdown. We will make a subsequent ticket to make the url host align with the environment.

File: content/home/dataset-cloud.md [hunk 2]
```diff
@@ -4,62 +4,46 @@ datasets:
   - title: "CMG"
     actions:
       - label: "Learn More"
-        url: "/overview/data-consortia#cmg"
-      - label: "Datasets"
-        url: "/data?query=consortium%3DCMG%252COR"
+        url: "https://anvilproject.org/data/consortia/CMG"
```
Comment (frano-m) [inline content/home/dataset-cloud.md:7]: The dataset cloud markdown now only links to the corresponding consortia detailed page e.g. https://anvilproject.org/data/consortia/CMG. I have removed the link for the datasets. We should make a ticket to make these links open in the same tab (current behaviour is a new tab is opened).

File: content/site-map.yaml [hunk 3]
```diff
@@ -36,17 +36,6 @@
 #    n.b. If "file" is an empty string, the navigation item is a "toggle" button where the partial will be omitted from any children navigationItems paths.
 # - "pathOverride" overrides and replaces any built paths from partials.
 
-## Datasets
```
Comment (frano-m) [inline content/site-map.yaml:39]: Removing datasets from the site-map for both NCPI and AnVIL, as well as any old roadmap references.

File: gatsby-node.js [hunk 4]
```diff
@@ -38,13 +38,12 @@ exports.onCreateNode = ({ actions, getNode, node }) => {
 
   if (type === "MarkdownRemark") {
     const { frontmatter } = node,
-      { pageAlignment, privateEvent } = frontmatter || {};
+      { privateEvent } = frontmatter || {};
```
Comment (frano-m) [inline gatsby-node.js:41]: Removal of deprecated pageAlignment here and elsewhere.

File: gatsby-node.js [hunk 5]
```diff
@@ -122,26 +116,6 @@ exports.createPages = ({ actions, graphql }) => {
           }
         }
       }
-      allDashboardStudy {
```
Comment (frano-m) [inline gatsby-node.js:125]: Removal of any dashboard/workspaces related graphql queries.

File: gatsby-node.js [hunk 6]
```diff
@@ -210,46 +179,6 @@ exports.createPages = ({ actions, graphql }) => {
         });
       }
     });
-
-    /* For each dashboard study create a detailed page. */
-    allDashboardStudy.edges.forEach(({ node }) => {
```
Comment (frano-m) [inline gatsby-node.js:215]: Removal of page creation for any dashboard related items.

File: src/components/article-content-container/article-content-container.js [hunk 7]
```diff
@@ -7,19 +7,14 @@
...
 import * as compStyles from "../article-content-container/article-content-container.module.css";
 
 function ArticleContentContainer(props) {
-  const { children, left } = props;
-  const classNamesContentContainer = classNames(compStyles.contentContainer, {
-    [compStyles.left]: left,
-  });
-
-  return <div className={classNamesContentContainer}>{children}</div>;
+  const { children } = props;
```
Comment (frano-m) [inline src/components/article-content-container/article-content-container.js:16]: Removal of now deprecated page alignment styles.

File: src/components/button-cta/button-cta.tsx [hunk 8]
```diff
@@ -15,13 +15,14 @@ import React, { FC, ReactNode } from "react";
...
 import ButtonCtaNavigationMethod from "./button-cta-navigation-method";
-import Target from "../target/target.model";
+import { Target } from "../target/target.model";
 
 // Styles
 import { button, large, primary, secondary } from "./button-cta.module.css";
 
 interface Props {
   attributeHREF: string;
+  attributeTarget?: Target;
```
Comment (frano-m) [inline src/components/button-cta/button-cta.tsx:25]: Added target an an optional prop to ButtaCTA component - allows the "Explore Datasets" button on the homepage to open in the same tab.

File: src/components/home/section-dataset-cloud/section-dataset-cloud.tsx [hunk 9]
```diff
@@ -58,7 +60,8 @@ const SectionDatasetCloud: FC = (): JSX.Element => {
             Consortia Roadmap
           </ButtonCta>
           <ButtonCta
-            attributeHREF="/data"
+            attributeHREF={`${getDatasetsEnvironmentUrl()}data/consortia/`}
```
Comment (frano-m) [inline src/components/home/section-dataset-cloud/section-dataset-cloud.tsx:63]: "Explore Datasets" now opens in the same tab, with the updated datasets url.

File: src/components/stats/stats.tsx [hunk 10]
```diff
@@ -6,8 +6,8 @@ const Stats: FC = (): JSX.Element => {
   return (
     <div className={compStyles.stats}>
       <Stat stat={500}>Cohorts</Stat>
-      <Stat stat={600}>Petabytes</Stat>
-      <Stat stat={4} unit="thousand">
+      <Stat stat={4}>Petabytes</Stat>
```
Comment (frano-m) [inline src/components/stats/stats.tsx:9]: Stats counts are corrected.

File: src/utils/node/create-pages.service.js [hunk 11]
```diff
@@ -9,7 +9,6 @@
 // Template variables
 const ComponentPath = {
   CONTENT: "src/templates/content-template.js",
-  DATASET_STUDY: "src/templates/dashboard-study-template.js",
```
Comment (frano-m) [inline src/utils/node/create-pages.service.js:12]: Removal of template! Woot! No longer needed!

**Review by NoopDog — APPROVED** [review]

**Pull request merged**